flexget/plugins/operate/verify_ssl_certificates.py

```diff
@@ -23,6 +24,16 @@ class VerifySSLCertificates(object):
     def on_task_start(self, task, config):
         if config is False:
             task.requests.verify = False
+            # Disabling verification results in a warning for every HTTPS
+            # request:
+            # "InsecureRequestWarning: Unverified HTTPS request is being made.
+            #  Adding certificate verification is strongly advised. See:
+            #  https://urllib3.readthedocs.io/en/latest/security.html"
+            # Disable those warnings because the user has explicitly disabled
+            # verification and the warning is not beneficial.
+            # This change is permanent rather than task scoped, but there won't
+            # be any warnings to disable when verification is enabled.
+            urllib3.disable_warnings()
```

Comment:
Should we be more specific about what warning to disable? i.e. `urllib3.disable_warnings(InsecureRequestWarning)`